packages/office-ui-fabric-react/src/components/Label/LabelCodepenButton.tsx

```diff
@@ -0,0 +1,36 @@
+import * as React from 'react';
+import { CommandButton } from '../..';
+import { css } from 'office-ui-fabric-react/lib/Utilities';
+
+const jsContent = require('!raw-loader!office-ui-fabric-react/src/components/Label/codepen_examples/Label.Basic.CodepenExample.txt') as string;
+const htmlContent = `<script src="//unpkg.com/office-ui-fabric-react/dist/office-ui-fabric-react.min.js"></script>
+<div id=\'content\'></div>`;
+
+const headContent = `<script type="text/javascript" src="https://unpkg.com/react@16/umd/react.development.js"></script><script type="text/javascript" src="https://unpkg.com/react-dom@16/umd/react-dom.development.js"></script>`;
```

Comment:
shouldn't this also import fabric? Where is fabric imported?

Reply:
I used the same boilerplate as the code used for the Fabric Starting Template (https://codepen.io/FabricReact/pen/NvBvWx), and it seems to work fine?